snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -149,10 +150,7 @@ func (b *Bootstrapper) Clear(context.Context) error {
 	b.Ctx.Lock.Lock()
 	defer b.Ctx.Lock.Unlock()
 
-	if err := b.Config.Blocked.Clear(); err != nil {
-		return err
-	}
-	return b.Config.Blocked.Commit()
+	return database.AtomicClear(b.DB, b.DB)
```

Comment:
We use atomic clear here to ensure we don't get into a position where the database has only been half deleted.

Reply:
I wonder if this comment should be in the codebase